plugins/jobs/src/js/jobsRunNow.js

```diff
@@ -0,0 +1,27 @@
+import "rxjs/add/operator/take";
```

Comment:
🎨 no `.ts` 😭 (this file)

Reply:
Also: shouldnt it be `jobRunNow`? 🤔 For clarification I'd love to see these files name `JobsMenuRunNow` or similar 🤔

Reply:
please look at the jobsDelete answer.